Implement bookmark's hooks for export etc.

jordandukart opened the pull request.

jordandukart committed:
Implement bookmark's hooks for export etc.

adam-vessey commented on usc_mirc.module:
```diff
@@ -1456,3 +1456,73 @@ function usc_mirc_islandora_solr_query($islandora_solr_query) {
+      if ($object['PBCORE']) {
+        $pb_xml = simplexml_load_string($object['PBCORE']->content);
+        $pb_xml->registerXPathNamespace('pb', 'http://www.pbcore.org/PBCore/PBCoreNamespace.html');
+        if (in_array('usc:vroCModel', $object->models)) {
```
Use the variable_get() dealio for the content model? (I'd store it in a local variable first).

adam-vessey commented on usc_mirc.module:
```diff
@@ -1456,3 +1456,73 @@ function usc_mirc_islandora_solr_query($islandora_solr_query) {
+        $pb_xml->registerXPathNamespace('pb', 'http://www.pbcore.org/PBCore/PBCoreNamespace.html');
+        if (in_array('usc:vroCModel', $object->models)) {
+          $pb_results = $pb_xml->xpath('pb:pbcoreInstantiation/pb:instantiationLocation/text()');
+          dsm($pb_results);
```
dsm()!... 'd

adam-vessey commented on usc_mirc.module:
```diff
@@ -1456,3 +1456,73 @@ function usc_mirc_islandora_solr_query($islandora_solr_query) {
+            }
+          }
+        }
+        elseif (in_array('usc:mezzanineCModel', $object->models)) {
```
Same dealio with the variable_get()...

adam-vessey committed:
Add missing dependency.

adam-vessey committed:
Add COLLECTION_POLICY to root collection.

Also, squash references to "Adding a subcollection," as it's now
handled through the collection solution pack.

jordandukart committed:
Code review fixes.
